javascript/javascript2/week3/keypresser-Game/main.js

```diff
@@ -0,0 +1,96 @@
+const jsConfetti = new JSConfetti()
+async function confettiDemo(){
+    await jsConfetti.addConfetti()
+    setTimeout(
+    function pageRelode(){
+    window.location.reload();
+    },2000)
+}
+const inputTime = document.getElementById('input-key');
+const btnStart = document.getElementById('btn-start');
+const btnStop = document.getElementById('btn-stop');
+const btnAgain = document.getElementById('btn-again');
+const countdown = document.getElementById('countdown')
+const sKey = document.getElementById('box-1')
+const lKey = document.getElementById('box-2')
+const resultBox = document.getElementById('box-3')
+const winnerResult = document.getElementById("winner");
+const skeyOutput = document.getElementById("skey")
+const lkeyOutput = document.getElementById("lkey")
```

Comment:
I would suggest creating the elements in the js from the start and then appending them if the style of the page allows it. The reason for that is when calling getElementById, you need to come up with unique Ids every time you add something in HTML. Also, it increases the possibility of errors or typos from the HTML to Js.